.readthedocs.yml

```diff
@@ -36,5 +36,7 @@ build:
 python:
   version: 3.8
   install:
+  - method: pip
```

Comment:
Let's add this to `docs/requirements.in` instead.

Reply:
I am confident that this may introduce problems to tox, dependabot, and possible even RTD as it would require a relative path ".." inside a reqs file. Lets keep it out so we can control how we install our package opposed to the docs requirements. Even RTD examples display that.

Reply:
@ssbarnea ^